lib/Controller/BackendController.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * @copyright Copyright (c) 2017 Joachim Bauch <bauch@struktur.de>
+ *
+ * @author Joachim Bauch <bauch@struktur.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Spreed\Controller;
+
+use OCA\Spreed\Config;
+use OCA\Spreed\Room;
+use OCP\AppFramework\Controller;
+use OCP\Http\Client\IClientService;
+use OCP\ILogger;
+use OCP\IRequest;
+use OCP\Security\ISecureRandom;
+
+class BackendController extends Controller {
```

Comment:
This controller has no routes, drop it?
Or you forgot to commit something?

Reply:
No longer a controller now.